### DISCUSSION TITLE
Ru store

### DISCUSSION THREAD

**Discussion by osiriso1** [jef/streetmerchant#2849] [Ideas and feature requests]
Hello, please add RU stores
https://store.sony.ru/playstation/playstation_5/console_playstation_5/
https://www.wildberries.ru/catalog/15298664/detail.aspx?size=44592460
https://www.1c-interes.ru/catalog/konsoli/igrovaya_konsol_playstation_5/?utm_medium=cpa&utm_source=1672766&utm_campaign=admitad&admitad_uid=b5b7251fd9a11436c3fa115b50a4f805
https://www.mvideo.ru/playstation-4327/ps5-konsoli-8627/f/category=sony-playstation-5-ps5-8628

**Comment by vadimkaxd** [comment]
Hi, give a telegram

**Comment by osiriso1** [comment]
@osiriso

чт, 10 февр. 2022 г., 02:57 Vadim ***@***.***>:
